pkg/controller/operators/olm/operator.go

```diff
@@ -1097,7 +1097,7 @@ func (a *Operator) syncClusterServiceVersion(obj interface{}) (syncError error)
 	}
 
 	// status changed, update CSV
-	if !(outCSV.Status.LastUpdateTime == clusterServiceVersion.Status.LastUpdateTime &&
+	if !(outCSV.Status.LastUpdateTime.Equal(clusterServiceVersion.Status.LastUpdateTime) &&
```

Comment:
Forgot that this could be a nil dereference if `outCSV.Status.LastUpdateTime == nil`. Going to add a check for that first.

Reply:
Actually, never mind. I keep forgetting a method is orthogonal to the underlying data and `LastUpdateTime.Equal()` is fine even if `LastUpdateTime == nil`.